lionblog-yeeun/src/routes/PostDetailPage.jsx

```diff
@@ -1,7 +1,33 @@
+import React, { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import posts from "../data/posts";
+import { BigPost } from "../components/Posts";
+
 const PostDetailPage = () => {
-	return (
-		<div>Post Detail Page</div>
-	);
-};
+  const { postId } = useParams();
+  const [post, setPost] = useState(null);
+  useEffect(() => {
+    const post = posts.find((post) => post.id === parseInt(postId));
+    setPost(post);
+  }, [postId]);
+  const handleDelete = () => {
+    alert("삭제"); // TODO: add api call for sign up
+  };
 
-export default PostDetailPage;
\ No newline at end of file
+  return (
+    <div className="w-5/6">
+      <div className="px-10 mt-10 flex justify-center">{post && <BigPost post={post} />}</div>
+      <div className="flex flex-row justify-center gap-5 mb-5">
+        <Link to="edit">
```

Comment:
세미나 자료 과제 설명 + App.js를 보시면, postEditPage는 /${id}/edit 과 같은 형식으로 들어가게 되어 있습니다! Link 태그에 post의 id를 전달해 보아요 ~
```suggestion
        <Link to={`/${post.id}/edit`}>
```